test/units/module_utils/facts/test_date_time.py

```diff
@@ -66,8 +65,8 @@ def fake_date_facts(fake_now):
         ('time', '12:34:56'),
         ('iso8601_basic', '20200711T123456124356'),
         ('iso8601_basic_short', '20200711T123456'),
-        ('iso8601_micro', '2020-07-11T02:34:56.124356Z'),
-        ('iso8601', '2020-07-11T02:34:56Z'),
+        ('iso8601_micro', '2020-07-11T12:34:56.124356Z'),
+        ('iso8601', '2020-07-11T12:34:56Z'),
```

Comment:
Why change these to match the local time?

I suspect they were set to different values to allow the test to verify the two were not being mixed up.

Reply:
Fixed.